packages/input_schema/src/input_schema.ts

```diff
@@ -101,9 +109,9 @@ const validateField = <T extends Record<string, any>> (validator: Ajv, fieldSche
         return;
     }
 
-    // If there are more matching definitions then the type is string and we need to get the right one.
+    // If there are more matching definitions then the type is string, and we need to get the right one.
     // If the definition contains "enum" property then it's enum type.
-    if (fieldSchema.enum) {
+    if ((fieldSchema as StringFieldDefinition).enum) {
```

Comment:
I was a bit indecisive here.

The right thing to do is to use discriminating union
```
if (fieldSchema.type === 'string' &&
```
but that could create a breaking change